collections/_article/introducing-new-forum.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Introducing the new Godot Forum"
+excerpt: "We are excited to introduce a major improvement in terms of funding the development efforts of the Godot Engine."
```

Comment:
```suggestion
excerpt: "We are finally unveiling the new Godot forums which replace our Q & A platform while unlocking many new features."
```

Reply:
Updated based on your comment. I think you are right and "Forum" makes more sense

Reply:
Don't forget the "s" on "replaces" :)